### ISSUE TITLE
Format of training dataset

### ISSUE THREAD

**Issue by cuongngm** [CrazySummerday/ctpn.pytorch#2]
Hi, your repo is very useful for me. I have a questions. Can you describe the format of training dataset? How look like structure of dataset prepare? The labels is .xml? I need train my own dataset, wait for your reply

**Comment by CrazySummerday** [comment]
Pascal VOC

**Comment by CrazySummerday** [comment]
> Hi, your repo is very useful for me. I have a questions. Can you describe the format of training dataset? How look like structure of dataset prepare? The labels is .xml? I need train my own dataset, wait for your reply

Pascal VOC

**Comment by cuongngm** [comment]
Can you help me a way to visualize training loss like your repo?

**Comment by cuongngm** [comment]
i solved problem by get epoch and the losses in the checkpoints folder. Close issue
